x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/resilient/case_mappings/utils.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { CasesConfigurationMapping } from '../types';
```

Comment:
It make sense to move all files, which is common for ServiceNow, Jira and Resilient to the common folder `case_mappings` to avoid the code duplication